### ISSUE TITLE
Component - Expose - Button has no accessible name

### ISSUE THREAD

**Issue by LIT-EIA-OWNER** [MeD-DMC/Adapt-accessibilityfixes#15]
Button to expose has no accessible name <button class="expose-item-button"></button>

button has no label so a screen reader user would not know the intent of the activity

**Comment by GGD4V** [comment]
Last 2 commits I added: 

- Aria attributes
- button role
- aria pressed toggle method
- key enter detection

Following patterns from https://www.w3.org/TR/wai-aria-practices-1.1/examples/button/button.html
